Password Generator/PasswordGenerator_UtkarshTiwari/script.js

```diff
@@ -0,0 +1,192 @@
+const inputSlider = document.querySelector("[data-lengthSlider]");
+const lengthDisplay = document.querySelector("[data-lengthNumber]");
+const passwordDisplay = document.querySelector("[data-passwordDisplay]");
+const copyBtn = document.querySelector("[data-copy]");
+const copyMsg = document.querySelector("[data-copyMsg]");

```

Comment:
What is the rationale behind using this custom attributes particularly in your app?